### ISSUE TITLE
Allow passing lockup_ids for unlocking

### ISSUE THREAD

**Issue by p-offtermatt** [informalsystems/hydro#197]
For vote aggregators, it's important that they can unlock only some locks, to avoid a user paying for unlocks on other users lockups.

We should consider whether to skip locks that are not actually expired, instead of erroring out.

**Comment by arlai-mk** [comment]
Should this issue be closed?

**Comment by p-offtermatt** [comment]
Yes! I think it was supposed to auto-close (from the PR) but maybe didn't because the PR came from a fork. I've closed manually
